src/core/Utils.cpp

```diff
@@ -399,6 +399,30 @@ QString extractHost(const QUrl &url)
 	return (url.isLocalFile() ? QLatin1String("localhost") : url.host());
 }
 
+QString extractDomainName(const QUrl &url)
+{
+	if (!url.isValid())
+	{
+		return {};
+	}
+
+	const QString host = url.host();
+	if (host.isEmpty())
+	{
+		return {};
+	}
+
+	const QString tld = url.topLevelDomain();
+	if (tld.isEmpty())
+	{
+		return {};
+	}
+
+	const int domainStartIdx = host.lastIndexOf('.', (-1 * tld.size()) - 1) + 1;
```

Comment:
'+1' at the end because:
* in case lastIndexOf does not find a dot, it will return -1, adding 1 will set the domainStartIdx to 0
* in case lastIndexOf does find a dot, it should return the idx after the dot

ie.
220.ro -> lastIndexOf returns -1, we add 1, domainStartIdx = 0, return value = 220.ro
vid.220.ro -> lastIndexOf returns 3, we add 1, domainStartIdx = 4, return value = 220.ro